lib/ecco/row_event_listener.rb

```diff
@@ -5,8 +5,11 @@ module Ecco
   class RowEventListener < EventListener
     ROW_EVENTS = [
       EventType::WRITE_ROWS,
+      EventType::EXT_WRITE_ROWS,
       EventType::UPDATE_ROWS,
+      EventType::EXT_UPDATE_ROWS,
       EventType::DELETE_ROWS,
+      EventType::EXT_DELETE_ROWS,
```

Comment:
Might be possible to configure [MySQL to use v1 ROWS_EVENT](http://dev.mysql.com/doc/refman/5.6/en/replication-options-binary-log.html#sysvar_log_bin_use_v1_row_events) but it feels better to support both v1 and v2.